dot/parachain/pvf/host.go

```diff
@@ -0,0 +1,148 @@
+package pvf
+
+import (
+	"fmt"
+	"sync"
+
+	parachainruntime "github.com/ChainSafe/gossamer/dot/parachain/runtime"
+	parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
+	"github.com/ChainSafe/gossamer/internal/log"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+)
+
+var logger = log.NewFromGlobal(log.AddContext("pkg", "pvf"), log.SetLevel(log.Debug))
+
+type ValidationHost struct {
+	wg     sync.WaitGroup
```

Comment:
No need for waitgroup here. Instead pass the waitgroup from candidate validation to the `Start` method